quic/s2n-quic/src/provider/endpoint_limits.rs

```diff
@@ -3,9 +3,13 @@
 
 //! Allows applications to limit peer's ability to open new connections
 
-pub use s2n_quic_core::endpoint::{
-    limits::{ConnectionAttempt, Outcome},
-    Limiter,
+pub use s2n_quic_core::{
+    endpoint::{
+        limits::{ConnectionAttempt, Outcome},
+        Limiter,
+    },
+    path::THROTTLED_PORTS_LEN,
+    time::Timestamp,
 };
```

Comment:
We need to preserve the public exports and separate the private imports:

```suggestion
pub use s2n_quic_core::endpoint::{
    limits::{ConnectionAttempt, Outcome},
    Limiter,
};
use s2n_quic_core::{
    path::THROTTLED_PORTS_LEN,
    time::Timestamp,
};
```